apps/expo/package.json

```diff
@@ -67,7 +67,9 @@
     "@stripe/stripe-js": "^1.52.1",
     "@tradle/react-native-http": "^2.0.1",
     "@types/tinycolor2": "^1.4.3",
+    "@walletconnect/react-native-compat": "^2.7.8",
     "@walletconnect/react-native-dapp": "^1.8.0",
+    "@web3modal/react-native": "^1.0.0-alpha.8",
```

Comment:
I saw this lib has been deprecated, do we still need it?
![CleanShot 2023-06-26 at 8 55 22](https://github.com/showtime-xyz/showtime-frontend/assets/37520667/1d5bd1ce-582b-44ed-9d07-1c3dcb6d3311)

Reply:
In Slack we received:

* @walletconnect/ethereum-provider:  2.8.3 or higher
* @walletconnect/modal: 2.5.4 or higher

@intergalacticspacehighway

Reply:
good catch. Let me remove this deps. We've forked it and using a custom version. Chatting with wallet connect team to add the features we need.